plugin/src/test/java/org/opensearch/ml/autoredeploy/MLModelAutoReDeployerIT.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.ml.autoredeploy;
+
+import static org.opensearch.ml.common.MLTask.MODEL_ID_FIELD;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.opensearch.ml.common.MLTaskState;
+import org.opensearch.ml.rest.MLCommonsRestTestCase;
+
+import lombok.SneakyThrows;
+
+public class MLModelAutoReDeployerIT extends MLCommonsRestTestCase {
+
+    public void testModelAutoRedeploy() {
+        prepareModel();
+    }
+
+    @SneakyThrows
+    private void prepareModel() {
```

Comment:
[nit] why do we need this method then?

Reply:
It can be removed but it's not an urgent issue. We can optimize in future PR